### PR TITLE
OEL-256: Add Facets Form and Facets Summary to Showcase Search

### PR DESCRIPTION
**Pull request by GilNovacomm**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by Maxfire — APPROVED** [review]
checked tests and func and it worked, only needs rebasing and solve conflicts
Greetings.

**Review by Maxfire — CHANGES_REQUESTED** [review]
rebase it, solve conflicts and push it again, the rest is tested.
Greetings.

**Review by cmarcmar — DISMISSED** [review]

File: modules/oe_showcase_search/config/install/facets.facet.category.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+min_count: 1
+show_only_one_result: false
+field_identifier: field_category
+facet_source_id: 'search_api:views_page__showcase_search__showcase_search_page'
+widget:
+  type: facets_form_checkbox
+  config:
+    disabled_on_empty: false
+    indent_class: indented
+    show_numbers: false
```
Comment (drishu) [inline modules/oe_showcase_search/config/install/facets.facet.category.yml:25]: enable term label by id

File: modules/oe_showcase_search/oe_showcase_search.info.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ dependencies:
   - search_api_db:search_api_db
   - drupal:views
   - openeuropa:oe_showcase_default_content
+  - facets_form:facets_form
+  - facets_summary:facets_summary
+  - facets_form_date_range:facets_form_date_range
```
Comment (drishu) [inline modules/oe_showcase_search/oe_showcase_search.info.yml:12]: above might be dependency of this, check and remove

**Review by drishu — CHANGES_REQUESTED** [review]
Lets place the summary and facets form blocks for now, as it makes more sense

File: modules/oe_showcase_search/oe_showcase_search.info.yml (hunk 2 above)
Comment (GilNovacomm) [reply modules/oe_showcase_search/oe_showcase_search.info.yml:12]: Facets form is indeed required for Form Date Range, for Summary only Facets is a dependency so I kept:
- facets_summary:facets_summary
- facets_form_date_range:facets_form_date_range

**Review by drishu — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
approved :+1:

**Pull request merged**